src/UiPopover.vue

```diff
@@ -113,12 +113,14 @@ export default {
                 return;
             }
 
+            const body = this.triggerEl.getRootNode() === document ? document.body : this.triggerEl.getRootNode();
```

Comment:
A comment explaining why we're doing this would be nice.

Reply:
Indeed. Added a comment.